### DISCUSSION TITLE
Ability to change all margin, padding etc properties together

### DISCUSSION THREAD

**Discussion by ravisaxena** [GrapesJS/grapesjs#5707] [Q&A]
Hello, right now grapesjs provides built-in properties like margin, and padding that allow users to change each property one by one (margin-top, margin-bottom, etc).

It would be nice to have the ability to change all these properties together. To have some lock button, for example, when it's enabled it will change all properties together, not one by one.

@artf Please help or suggest any alternative. Most of the builders have that feature. For usability prospects it is beneficial

**Comment by artf** [comment]
Please read the documentation first https://grapesjs.com/docs/modules/Style-manager.html
